### ISSUE TITLE
Support default_value in make_agg_primitive

### ISSUE THREAD

**Issue by kmax12** [FeatureLabs/featuretools#55]
If you are using the `make_agg_primitive` to define a custom primitive, you currently cannot provide the `default_value` to use. This parameter is supported by the lower level class-based definition of primitives, so it should available through the helper method.

**Comment by Seth-Rothschild** [comment]
It seems like `make_agg_primitive` currently decides what the default value is by passing in empty data to the provided function:
```
# infers default_value by passing empty data
try:
    new_class.default_value = function(*[[]] * len(input_types))
except Exception:
    pass
```
We could just change that to `new_class.default_value = default_value` and we set a default as a kwarg?

**Comment by kmax12** [comment]
Good catch.

I don't think we need to add the kwarg, but we should put this somewhere in the documentation and doc string for the function.

**Comment by kmax12** [comment]
reopening. a user should be able to specify the default value when creating the primitive to avoid the inference step. if no default value if provided, the inference should happen. 

for example, 
```
Min = make_agg_primitive(
    np.min,
    [Numeric],
    Numeric,
    name="Min",
    stack_on_self=False,
    default_value=None,
    description="Finds the minimum non-null value of a numeric feature.")
```

**Comment by bukosabino** [comment]
To finish this issue, I would like to see an example of default_value that make sense.

Can someone provide me based on this example primitive?

```
def maximum(column):
    return max(column)

Maximum = make_agg_primitive(function=maximum,
                            input_types=[Numeric],
                            return_type=Numeric,
                            default_value=[2,2,2,2])
```

Thanks

**Comment by kmax12** [comment]
if the maximum function you define is an aggregation primitive, then it will always return a since value that is the largest in a list of numbers. the default value should be the return value when the input list is empty. in this case, that should be `numpy.nan`. the default value should never be a list, always a single value

**Comment by rwedge** [comment]
Fixed by #479
